etc/picongpu/lumi-eurohpc/standard-g.tpl

```diff
@@ -118,6 +113,26 @@ else
     run_cuda_memtest=0
 fi
 
+
+cat << EOF > select_gpu
+#!/bin/bash
+
+env
```

Comment:
I introduced this line. It is too much output in most cases. I will open up a followup pull request to remove it.